src/components/Modals/Send/hooks/useSendDetails/useSendDetails.test.tsx

```diff
@@ -237,15 +234,31 @@ describe('useSendDetails', () => {
       expect(result.current.fieldName).toBe('fiat.amount')
 
       // Set fiat amount
-      act(() => {
+      await act(async () => {
         result.current.handleInputChange('3500')
+        await new Promise(r => setTimeout(r, 1500)) // hack to wait because handleInputChange is now debounced for 1 second
```

Comment:
You can use mock timers instead from the jest library.

Reply:
I tried messing with that and couldnt get it to work. I googled and read some people saying they had problems with doing that with lodash debounce

Reply:
have a look at useBalanceChartData.test.ts for how to do it. this adds at least 3 seconds to the already slow tests and will be copied as a pattern making them even slower

Reply:
I think its different because those tests are for a custom debounce function that uses setTimeout instead of the lodash debounce.

I tried messing with it again a bit and couldn't make it work.

Maybe removing the tests is another option if we are worried about it slowing down ci